docs/usage/self-hosted-configuration.md

```diff
@@ -7,6 +7,10 @@ description: Self-Hosted Configuration usable in renovate.json or package.json
 
 The below configuration options are applicable only if you are running your own instance ("bot") of Renovate.
 
+## allowMergeWhenPipelineSucceeds
+
+When this option is enabled it is possible that MRs with failing pipelines are getting merged. This is caused by a race condition in GitLab's Merge Request API - [read the issue](https://gitlab.com/gitlab-org/gitlab/issues/26293) for details.
```

Comment:
Can you also add some text about what it is supposed to do? I agree that the warning is necessary, but a sentence or two about the successful behaviour might be useful as well.

Reply:
The happy case is explained at https://github.com/renovatebot/renovate/pull/4558/files#diff-65aa9acc5e40f0995fc4d4ab67a2eeafR2004. The text here is just appended (please correct me when I got this wrong @rarkins).